### PR TITLE
Access control: add roles to fixed groups

### PR DESCRIPTION
**Pull request by IevaVasiljeva**
**What this PR does / why we need it**:
This PR adds groups to fixed roles. This will be used by role picker to display roles in sections.

**Which issue(s) this PR fixes**:

Partially fixes https://github.com/grafana/grafana-enterprise/issues/2168

**Special notes for your reviewer**:
Changes for enterprise roles are here: https://github.com/grafana/grafana-enterprise/pull/2274

### CODE REVIEW

File: pkg/services/accesscontrol/roles.go [hunk 1]
```diff
@@ -74,7 +79,8 @@ var (
 
 	usersOrgReadRole = RoleDTO{
 		Name:    usersOrgRead,
-		Version: 1,
+		Version: 2,
+		Group:   "Organization users",
```
Comment (sakjur) [inline pkg/services/accesscontrol/roles.go:83]:
```suggestion
		Group:   "User administration (organizational)",
```

File: pkg/services/accesscontrol/roles.go [hunk 2]
```diff
@@ -104,7 +111,8 @@ var (
 
 	usersAdminReadRole = RoleDTO{
 		Name:    usersAdminRead,
-		Version: 1,
+		Version: 2,
+		Group:   "Users",
```
Comment (sakjur) [inline pkg/services/accesscontrol/roles.go:115]:
```suggestion
		Group:   "User administration (global)",
```

**Review by sakjur — COMMENTED** [review]
I don't know how to convey the power of the users endpoints. I'm a little bit concerned that "users" doesn't convey the power that comes with those endpoints (namely that you'd be able to give a user access to any org).

**Comment by IevaVasiljeva** [comment]
> I don't know how to convey the power of the users endpoints. I'm a little bit concerned that "users" doesn't convey the power that comes with those endpoints (namely that you'd be able to give a user access to any org).

That's a good point. I quite like the sound of `User administration`. What do you think @osg-grafana, @kalleep, @mjseaman?

**Comment by IevaVasiljeva** [comment]
On a similar note - I also don't really like `Organizations` as one group, as it doesn't separate between global org roles (including creating and deleting orgs) and roles granting permissions only in the current org. What do others think?

**Review by gamab — APPROVED** [review]
Can't wait to see this on the frontend :)

**Comment by gamab** [comment]
> On a similar note - I also don't really like `Organizations` as one group, as it doesn't separate between global org roles (including creating and deleting orgs) and roles granting permissions only in the current org. What do others think?

These roles are really special, for them to be global or local rely solely on the assignment. If you assign it globally to Server Admins or to a user, the role will work on all orgs. If you assign it globally to an org role or locally to a user, the role will only work on the org you are signed into. So I think it's ok to have them in the same group. But I think the names I gave them could be improved...

**Review by osg-grafana — APPROVED** [review]

**Review by vtorosyan — APPROVED** [review]

**Pull request merged**